pkgs/by-name/on/onthespot/package.nix

```diff
@@ -32,7 +36,7 @@ python3.pkgs.buildPythonApplication rec {
 
   nativeBuildInputs = with python3.pkgs; [
```

Comment:
`with python3.pkgs;` is not needed, both `copyDesktopItems` and `kdePackages` are already defined